src/imports/background/__mocks__/item-processor.js

```diff
@@ -0,0 +1,14 @@
+export default class {
```

Comment:
Sure you don't a name here?

Reply:
Default exports don't require names (as there's only one per module). If added here, I think it would just be relevant in the scope of this module (like for accessing static props or creating instances of this class). Although I should probably just add one to be consistent, and put stuff like the setTimeout time (still having issues working with jest fake timers; not sure if will keep it like this) as static props.